backend/main.py

```diff
@@ -51,6 +51,7 @@ def __init__(self) -> None:
         if CONFIG["chown_plugin_path"] == True:
             self.web_app.on_startup.append(chown_plugin_dir)
         self.loop.create_task(self.loader_reinjector())
+        self.loop.create_task(self.plugin_reloader())
```

Comment:
All of this should be rewritten to use inotify, like the plugin hot-reload feature does